specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ManagedEnvironments.json

```diff
@@ -501,24 +768,119 @@
           "x-ms-secret": true
         }
       }
-    }
-  },
-  "securityDefinitions": {
-    "azure_auth": {
-      "type": "oauth2",
-      "description": "Azure Active Directory OAuth2 Flow",
-      "flow": "implicit",
-      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
-      "scopes": {
-        "user_impersonation": "impersonate your user account"
+    },
+    "Certificate": {
+      "description": "Certificate used for Custom Domain bindings of Container Apps in a Managed Environment",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Certificate resource specific properties",
+          "type": "object",
+          "properties": {
+            "password": {
+              "description": "Certificate password.",
+              "type": "string",
+              "x-ms-mutability": [
+                "create"
+              ],
+              "x-ms-secret": true
+            },
+            "name": {
+              "description": "Friendly name of the certificate.",
+              "type": "string"
+            },
```

Comment:
If this is the same name that is used in the URL of the API then it should not be here because it is a duplication of the name in the ARM envelope.

Reply:
Yes. It is the same. Removed it.